packages/tradershub/src/features/cfd/screens/CFDCompareAccounts/CompareAccountsButton.tsx

```diff
@@ -94,15 +95,14 @@ const CompareAccountsButton = ({ isAccountAdded, platform, shortCode }: TCompare
 
     const onClickAdd = () => {
         if (platform === CFDPlatforms.MT5) {
-            // Going to remove Placeholder once the MT5PasswordModal
-            // and Verification flow is implemented
+            // Going to remove Placeholder once Verification flow is implemented
```

Comment:
add TODO: infront of this message

Reply:
Adding TODO will give a warning from Sonarcloud